### PR TITLE
Fix errors on dbot test run 

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1883
Desc: This pr is to fix error on Quick Strategy and to update Feed verification code

### CODE REVIEW

File: cypress/e2e/smoke/dbot/FeedVerification.cy.js [hunk 1]
```diff
@@ -21,18 +21,22 @@ describe('QATEST-99340: Verify feed is loading on charts tab', () => {
...
-      charts.verifyTickChange(5000)
-
-      //check feed on demo account
-      cy.c_switchToDemoBot()
-      charts.openChartsTab()
-      charts.selectSymbolOnCharts('AUD/JPY')
-      charts.verifyTickChange(5000)
-      charts.selectSymbolOnCharts('Gold/USD')
-      charts.verifyTickChange(5000)
+      cy.findByTestId('dt_initial_loader').should('not.exist')
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/dbot/FeedVerification.cy.js:24]: We've a common custom command `c_loadingCheck` for this.

File: cypress/support/pageobjects/dbot/quick_strategy.js [hunk 2]
```diff
@@ -50,13 +50,23 @@ class QuickStrategy {
...
+          cy.findByText('Matches/Differs').click()
+        })
+    } else {
+      cy.findAllByTestId('dt_themed_scrollbars')
+        .eq(3)
+        .should('be.visible')
+        .within(() => {
+          cy.findByText('Matches/Differs').click()
+        })
+    }
```
Comment (abuzar-deriv) [inline cypress/support/pageobjects/dbot/quick_strategy.js:69]:
```suggestion
  chooseTradeType = (isMobile = false) => {
  const index = isMobile ? 1 : 3
      cy.findAllByTestId('dt_themed_scrollbars')
        .eq(index)
        .should('be.visible')
        .within(() => {
          cy.findByText('Matches/Differs').click()
        })
   }
```

File: cypress/e2e/smoke/dbot/QuickStrategies.cy.js [hunk 3]
```diff
@@ -38,14 +37,11 @@ describe('QATEST-4212: Verify Quick Strategy from bot builder page', () => {
         'Volatility 100 (1s) Index'
       )
       cy.findByTestId('qs_autocomplete_tradetype').click()
-      if (isMobile) {
-        cy.findByText('Matches/Differs', { exact: false }).first().click()
-      } else {
-        quickStrategy.chooseTradeType()
-      }
+      quickStrategy.chooseTradeType(size)
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/dbot/QuickStrategies.cy.js:40]:
```suggestion
      quickStrategy.chooseTradeType(isMobile)
```

**Review by abuzar-deriv — COMMENTED** [review]
Some suggestions...

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**